thisfairsoft.sh

```diff
@@ -54,6 +54,8 @@ fairsoft_repo() {
 fairsoft_repo "fairsoft_backports" "fairsoft-backports"
 fairsoft_repo "fairsoft" "fairsoft"
 
+spack clean -ms
+
```

Comment:
Please also add to `thisvae.sh`.

I did not factor out this, because we're still at two files. But if this becomes a maintainance issue, we should refactor.

Reply:
Done, see #298